packages/oidc-client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@aries-framework/oidc-client",
```

Comment:
Thinking whether oidc-client is unique enough. OIDC is something on it's own, and OIDC4VCI and OIDC4VP are specifically for SSI. We're not building a generic oidc client in this case. Not sure what a good name would be though

Reply:
Yeah I agree with this. I prefer to be a bit too verbose/explicit than incorrect. How would all the packages be named?

OIDC4VCI-(client|server)
OIDC4VP-(client|server)

So 4 packages?

Reply:
I think it'll be two packages. One for client (issuance + proving) and one for server (issuance + verification). That feels like the right tradeoff (hard to include server components in the client), but also keep more simplicity (instead of having 4 packagers).

Would you lean towards 4 packages?

Reply:
No my preference is for two packages (100% separate client and server). Just more for the naming as we do not want `OIDC-client` but `OIDC4VCI-client` is incorrect as it also does VP.

`OIDC4VCIVP-client` is getting a bit "odd". 

Context of `@aries-framework/` might imply enough that it is about the VCI/VP flow, but hard to say.

Reply:
Yes I was thinking the same. It really depends on where the VP stuff will go. That's why I kept it generic for now. I could change it to `oidc4vc-client` to at least make it VC-specific. Thoughts?

Reply:
Hmmm maybe `oidc4v(c|p)`? 😆 

But oidc4vc sounds good I think.

Reply:
Based on the comment below I have changed the package name to `@aries-framework/openid4vc-client`